src/Engine/Renderer/RenderTechnique/ShaderConfiguration.hpp

```diff
@@ -78,6 +78,7 @@ class RA_ENGINE_API ShaderConfiguration final
                          const std::string& vertexShader,
                          const std::string& fragmentShader );
 
+    ///\todo add shader from source.
```

Comment:
what about this todo?

Reply:
Well, it's a todo, maybe one day we will do it ... for now we have a "hack"

Reply:
As I do not know the "hack", Well, we need it really. I'll add that in PR #476.